scripts/okta/okta-login.ts

```diff
@@ -30,11 +30,36 @@ import { getRedirectUrl } from './lib/helper';
 // - /oauth2/<authorization server id>/v1/authorize
 // each will have a different query string and different config objects which are handled as appropriate in helper.ts.
 
-const redirectUrl = getRedirectUrl(
-	window.location.search,
-	window.location.origin,
-	window.location.pathname,
-	window.OktaUtil,
-);
-
-window.location.replace(redirectUrl);
+const locationSearch = window.location.search;
+const searchParams = new URLSearchParams(locationSearch);
+
+if (
+	searchParams.has('idp') ||
+	window.location.pathname === '/oauth2/v1/authorize/callback'
```

Comment:
Perhaps this condition needs some more explanation or a link to an explanation

Reply:
Added in a comment above the if statement!